src/constants/index.js

```diff
@@ -1,37 +1,43 @@
 export const SupportedNetwork = Object.freeze({
   MAINNET: "Mainnet",
   XDAI: "xDAI",
-  ARBITRUM_ONE: "Arbitrum",
+  ARBITRUM: "Arbitrum",
```

Comment:
Wondering why this got changed to only Arbitrum

Reply:
you are right I changed it back there was no need for that. I just needed to structure subgraph urls object and now it works!